testing/client_app/pages/landing/components/landing_component.py

```diff
@@ -0,0 +1,37 @@
+
+from client_app.pages.login.log_in_page import LogInPage
+from framework.abstract.component_base import BaseComponent
+from framework.factory.models_enums.constants_enums import LocatorType
+from selenium import webdriver
+from framework.factory.models_enums.page_config import ComponentModel
+
+
+class LandingComponent(BaseComponent):
+    """
+    This class is a component of the landing page.
+        It handles all events and UI locators of this entity.
+    """
+
+    def __init__(self, driver: webdriver, component_data: ComponentModel):
+        super(LandingComponent, self).__init__(driver, component_data)
+        self.btn_sign_in = self.get_locator(LocatorType.XPath, "//a[@class='Btn Login__button']")
+
+    def load_log_in_page(self) -> LogInPage:
+        """
+        Performs navigation to login page
+
+        Returns: instance of LoginPage class
+
+        """
```

Comment:
Please review the google styleguide to see examples of this format of docstrings as previously indicated (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods). 

The short description should be on the same line as the first `"""` and additional description if needed below after an additional newline. 

Content for Returns and Args should be indented below each.

```suggestion
        """Performs navigation to login page

        Returns: 
            instance of LoginPage class

        """
```